homeassistant/components/fritz/device_tracker.py

```diff
@@ -14,13 +14,14 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-CONF_DEFAULT_IP = "169.254.1.1"  # This IP is valid for all FRITZ!Box routers.
+DEFAULT_HOST = "169.254.1.1"  # This IP is valid for all FRITZ!Box routers.
+DEFAULT_USERNAME = "admin"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(CONF_HOST, default=CONF_DEFAULT_IP): cv.string,
-        vol.Optional(CONF_PASSWORD, default="admin"): cv.string,
-        vol.Optional(CONF_USERNAME, default=""): cv.string,
+        vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
+        vol.Optional(CONF_USERNAME, default=DEFAULT_USERNAME): cv.string,
+        vol.Optional(CONF_PASSWORD): cv.string,
```

Comment:
This may also be considered a breaking change

Reply:
I have only corrected the unnecessary default value of an empty string for `CONF_PASSWORD` here. Furthermore, the default value for password in fritzconnection.lib.fritzhosts is `None`, see the constructor of the fritz base class [here](https://github.com/kbr/fritzconnection/blob/516ddc2411712f9f59359908ec2d18f84f8fd4f9/fritzconnection/lib/fritzbase.py#L16). So it would not make a difference if dict.get() fails since it returns `None` by default.

Reply:
Ah, yeah I think you got it with the edit. If a user had the password set to `admin` and didn't specify the password, then this would break their config.

Reply:
> If a user had the password set to `admin` and didn't specify the password, then this would break their config.

Yes, exactly. Though I am not sure in which version of FritzOS that would be the case. Usually you are just given a password for the default "admin" user. That password is set by the manufacturer and should never be automatically set to "admin". Therefore, I am assuming there was a mix up of default username and password when this integration was created.